stdlib/src/collections/dict.mojo

```diff
@@ -899,6 +903,12 @@ struct Dict[K: KeyElement, V: CollectionElement](
         access the key and value as attributes ie.
 
         ```mojo
+        from collections import Dict
+
+        var dict = Dict[String, Int]()
+        dict["a"] = 1
```

Comment:
Nit: Could you replace `dict` with `d`? `dict` causes incorrect syntax highlighting on GH. Maybe also remove `var.

Reply:
Highlighting looks fine for me.
Also, I try to avoid single letter names, would using `my_dict` work?

Reply:
<img width="263" alt="Screenshot 2024-10-18 at 19 20 39" src="https://github.com/user-attachments/assets/e53ebde7-8fb2-4286-beac-3de0bf1b39c9">
`dict` will be blue because Python.

`my_dict` works.

Reply:
Do you mind changing this and the example below per @soraros feedback?

Reply:
I just changed this internally so we can land it.